tests/fixtures.py

```diff
@@ -20,15 +11,17 @@
 except ImportError:
     from itertools import izip_longest as zip_longest
 
-from datetime import datetime, timedelta
-day = timedelta(days=1)
-year = 365 * day                # close enough
-now = datetime.now()
 
 from caramel import (
     models,
     views,
-    )
+)
+
+from datetime import datetime, timedelta
```

Comment:
datetime import should be  high up as it's an python standardlib import.  

import order should be:

stdlib  then   exeternal modules last  internal modules